ipc-schema/facades/MobileSystemFacade.json

```
@@ -7,17 +7,68 @@
 	"methods": {
 		"findSuggestions": {
 			"doc": "Find suggestions in the OS contact provider.",
-			"arg": [{ "query": "string" }],
+			"arg": [
+				{
+					"query": "string"
+				}
+			],
 			"ret": "List<NativeContact>"
 		},
+		"saveContacts": {
+			"doc": "Store a single contact in system's contact book",
```

```suggestion
			"doc": "Store one or more contacts in system's contact book",
```
The name `saveContacts` & the arguments implies multiple contacts can be stored but the doc string says only a single can.